Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+                }
+
+                it("doesn't execute callback function twice") {
+                    eventDispatcher.once(eventName, callback: callback)
+
+                    eventDispatcher.trigger(eventName)
+                    callbackWasCalled = false
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == false
+                }
+
+                it("doesn't execute callback function when it is removed") {
+                    let listenId = eventDispatcher.once(eventName, callback: callback)
+                    eventDispatcher.off(listenId)
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == false
+                }
+            }
+
+            describe("listenTo") {
+                it("executes callback function for an event on a given context object") {
+                    let anotherDispatcher = EventDispatcher()
+
+                    eventDispatcher.listenTo(anotherDispatcher, eventName: eventName, callback: callback)
+                    anotherDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == true
+                }
+            }
+
+            describe("listenToOnce") {
+                it("executes callback function just one time for an event on a given context object") {
+                    let anotherDispatcher = EventDispatcher()
+
+                    eventDispatcher.listenToOnce(anotherDispatcher, eventName: eventName, callback: callback)
+                    anotherDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == true
+
+                    callbackWasCalled = false
+                    anotherDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == false
+                }
+            }
+
+            describe("off") {
+                it("doesn't executes callback function if it is removed") {
+                    let listenId = eventDispatcher.on(eventName, callback: callback)
+                    eventDispatcher.off(listenId)
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == false
+                }
+                it("doesn't execute callback if it is removed, but the others are called") {
+                    var anotherCallbackWasCalled = false
+                    let anotherCallback: EventCallback = { _ in
+                        anotherCallbackWasCalled = true
+                    }
+
+                    let listenId = eventDispatcher.on(eventName, callback: callback)
+                    eventDispatcher.on(eventName, callback: anotherCallback)
+
+                    eventDispatcher.off(listenId)
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == false
+                    expect(anotherCallbackWasCalled) == true
+                }
+            }
+
+            describe("stopListening") {
```

Comment:
According to the Project conventions, we should add the # prefix on the Method Name.

``describe("#stopListening")``